README.md

```diff
@@ -5,8 +5,10 @@
 | [pyribs.org](https://pyribs.org) | [GitHub](https://github.com/icaros-usc/pyribs) | [![PyPI](https://img.shields.io/pypi/v/ribs.svg?style=flat-square&color=blue)](https://pypi.python.org/pypi/ribs) | [![Conda Recipe](https://img.shields.io/badge/recipe-pyribs-green.svg?style=flat-square)](https://anaconda.org/conda-forge/pyribs) | [![Tests](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2Ficaros-usc%2Fpyribs%2Fbadge&style=flat-square)](https://github.com/icaros-usc/pyribs/actions?query=workflow%3A"Tests") | [docs.pyribs.org](https://docs.pyribs.org) | [![Documentation Status](https://readthedocs.org/projects/ribs/badge/?version=stable&style=flat-square)](https://readthedocs.org/projects/ribs/) | [![Twitter](https://img.shields.io/badge/twitter-%231DA1F2.svg?&style=flat-square&logo=twitter&logoColor=white)](https://twitter.com/pyribs) |
 
 A _bare-bones_ Python library for quality diversity optimization. pyribs is the
-official implementation of the Covariance Matrix Adaptation MAP-Elites (CMA-ME)
-algorithm and implements the _Rapid Illumination of Behavior Space (RIBS)_
+official implementation of Covariance Matrix Adaptation MAP-Elites (CMA-ME),
+Covariance Matrix Adaptation MAP-Elites via a Gradient Arborescence (CMA-MEGA),
+Covariance Matrix Adaptation MAP-Annealing (CMA-MAE), and scalable variants of
+CMA-MAE. pyribs implements the _Rapid Illumination of Behavior Space (RIBS)_
 redesign of MAP-Elites detailed in the paper
 [Covariance Matrix Adapation for the Rapid Illumination of Behavior Space](https://arxiv.org/abs/1912.02400).
```

Comment:
I think we either link all the papers here or don't link any of the papers -- to keep it consistent

Reply:
We link to CMA-ME since it has the original design, then we link all the papers below